src/søknad/steg/4-barnasbosted/BarnetsBostedEndre.tsx

```diff
@@ -116,27 +125,56 @@ const BarnetsBostedEndre: React.FC<Props> = ({
     })
     .filter(Boolean) as IBarn[];
 
+  const barnMedBarnetsForeldre = barn.forelder?.ident?.verdi
+    ? forelderidenterMedBarn.get(barn.forelder?.ident?.verdi)
+    : [];
+
+  const typeBarn =
+    barnMedBarnetsForeldre && barnMedBarnetsForeldre.length > 1
+      ? barnMedBarnetsForeldre.findIndex((b) => b.id === barn.id) === 0
+        ? TypeBarn.FØRSTE_AV_FLERE_BARN_MED_SAMME_FORELDER
+        : TypeBarn.BARN_MED_KOPIERT_FORELDER
+      : TypeBarn.ENESTE_BARN;
+
+  const [barnHarSammeForelder, settBarnHarSammeForelder] = useState<
+    boolean | undefined
+  >(
+    typeBarn === TypeBarn.ENESTE_BARN
+      ? barn.lagtTil && barn.annenForelderId === 'annen-forelder'
```

Comment:
Skulle gjerne hatt variabel eller funksjonsnavn for denne linja. Skjønner sånn halvveis hva som menes, men det er fordi jeg har sittet med søknaden nå nylig 😂

Reply:
Dette innførte en ny bug, så fjernet det.